islandora_solr_config/includes/rss_results.inc

```diff
@@ -191,7 +191,53 @@ class IslandoraSolrResultsRSS extends IslandoraSolrResults {
     $result['url'] = $base_url;
     $result['description'] = t('Aggregated search results of: @query', array('@query' => $query));
     $result['langcode'] = NULL;
+
+    $feed_url = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on' ? "https" : "http") . "://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]";
+    if (strpos($feed_url, '&page') == FALSE) {
+      $page_number = '1';
+    }
+    else {
+      $arr = explode('&page=', $feed_url);
+      $page_number = $arr[1];
+    }
+    $link_self = $feed_url;
+    if (isset($arr)) {
+      $link_first = $arr[0];
+    }
+    else {
+      $link_first = $feed_url;
+    }
+    $page_next = $page_number + 1;
+    if ($page_number > 1) {
+      $page_prev = $page_number - 1;
+    }
+    $link_next = url($link_first . '&page=' . $page_next);
```

Comment:
I'd like the query parameters (page) part of this to be done using the `query` argument to the `url` function instead of string concatenation.